config.php-default

```diff
@@ -16,3 +16,4 @@ $TIMEOUT = ;
 $BASE_URL = '';
 $MARKETPLACE_ID_FOR_OPERATOR = '' ;
 $SCRIPT_LOGIN = '';
+$SCRIPT_LOGIN_AGENT = '';
```

Comment:
Please, if first one not used. Remove it

Reply:
fixed